### ISSUE TITLE
pg transition: /people/activate/hashedId GET endpoint

### ISSUE THREAD

**Issue by JMurphyWeb** [emfoundation/ce100-app#444]
Endpoint: /people/activate/hashedId GET

Hoping to also remove iron and replace with jwt for the hashed id #132

**Comment by JMurphyWeb** [comment]
after converting the handler, i can see we need to use a people.getById(userId, cb) function:
```js
  request.server.methods.pg.people.getById(userId, function (pgErr, user) {
    Hoek.assert(!pgErr, 'database error');

    // if no user is found
    if (user.length === 0) {
      return reply(Boom.notFound('Something went wrong - we could not find your details'));
    }

    // if user has already activated
    if (user[0].account_activated) {
      return reply.redirect('/login');
    }

    return reply.view('activate', {error}).code(error ? 401 : 200);
```

**Comment by JMurphyWeb** [comment]
There is a slight problem with testing due to mock data: None of our mock users have `activated_account: false`.

Should I add a new primary user linked to one of the unlinked orgs (id 6 or 7) or should i change one of the current users that does not have an org and has active_true?

OR

Given that I have to add the data in 4 times, I am wondering if i should design my own data for this unit test and not load in the existing JSON files?

**Comment by SimonLab** [comment]
It might be a bit complicated to load your own data for a specific test because we don't link the data to the config object yet. I would just create a new user.

**Comment by mk4111** [comment]
I'm having a similar issue so I'm really glad you posted this!!

I think maybe creating a new user, but adding them to an existing unlinked org might give us a better way of testing? Only because I realise that we previously added a bunch of data which wasn't particularly significant of anything.

**Comment by mk4111** [comment]
I'm going to create a separate issue to mark the relationships and pieces of data that have been particularly important for me in testing challenges endpoints!

**Comment by JMurphyWeb** [comment]
Ok I will add a new user linked to org 6 and pr it into all repos

**Comment by JMurphyWeb** [comment]
@mk4111 slight change of plan: https://github.com/emfoundation/ce100-app/issues/447

**Comment by SimonLab** [comment]
resolved with #445
